### PR TITLE
Romanchuk/Delete city, region, district from address

### PR DESCRIPTION
**Pull request by alrom**

### CODE REVIEW

File: OutOfSchool/OutOfSchool.WebApi/Controllers/V1/StatisticController.cs [hunk 1]
```diff
@@ -33,19 +33,19 @@ public StatisticController(IStatisticService service)
...
-    /// <param name="city">City to look for.</param>
+    /// <param name="cATOTTGId">Codeficator's id.</param>
     /// <returns>List of popular directions.</returns>
     [HttpGet]
     [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<DirectionStatistic>))]
     [ProducesResponseType(StatusCodes.Status204NoContent)]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     [AllowAnonymous]
-    public async Task<IActionResult> GetDirections(int limit, [FromQuery] string city)
+    public async Task<IActionResult> GetDirections(int limit, [FromQuery] long cATOTTGId)
```
Comment (DmyMi) [inline OutOfSchool/OutOfSchool.WebApi/Controllers/V1/StatisticController.cs:43]: `cATOTTGId` probably would look better as `catottgId` if used as a method parameter

File: OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs [hunk 2]
```diff
@@ -30,7 +19,10 @@ public class AddressDto
 
     public double Longitude { get; set; }
 
-    public long? CATOTTGId { get; set; } = Constants.DefaultCityCodeficatorId;
+    [Required(ErrorMessage = "CATOTTGId is required")]
+    public long CATOTTGId { get; set; }
+
+    public CATOTTG CATOTTG { get; set; }
```
Comment (DmyMi) [inline OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs:25]: `AllAddressPartsDto CodeficatorAddressDto` should cover most of the codeficator already.

File: OutOfSchool/OutOfSchool.WebApi/Controllers/V1/StatisticController.cs (hunk 1 above)
Comment (alrom) [reply OutOfSchool/OutOfSchool.WebApi/Controllers/V1/StatisticController.cs:43]: Done

File: OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs (hunk 2 above)
Comment (alrom) [reply OutOfSchool/OutOfSchool.WebApi/Models/AddressDto.cs]: Deleted

**Review by DmyMi — APPROVED** [review]

**Pull request merged**